frontend/src/components/profile/components/user-info/user-info.tsx

```diff
@@ -22,6 +26,13 @@ type Props = {
 
 const UserInfo: React.FC<Props> = ({ user, isDoctor, onEdit }) => {
   const birthdate = getFormattedDate(user.birthdate, DateFormat.D_MMMM_YYYY);
+  const dispatch = useDispatch();
+
+  const handleUploadFile = (event: InputChangeEvent) => {
+    const file = (event.target.files as FileList)[DEFAULT_FILE_IDX];
```

Comment:
```suggestion
  const handleUploadFile = (evt: InputChangeEvent) => {
    const file = (evt.target.files as FileList)[DEFAULT_FILE_IDX];
```
Never user `event` as a variable name